django/db/backends/postgresql/features.py

```diff
@@ -84,6 +84,29 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         },
     }
 
+    @cached_property
+    def uses_server_side_binding(self):
+        return (
+            is_psycopg3
+            and self.connection.settings_dict["OPTIONS"].get("server_side_binding")
+            is True
+        )
+
+    @cached_property
+    def django_test_expected_failures(self):
+        excepted_failures = set()
+        if self.connection.use_server_side_binding:
+            # Parameters passed to expressions in SELECT and GROUP BY clauses
```

Comment:
I would move the comment above ""aggregation.tests.AggregateTestCase." in case more items are added later.